Guessing.js

```diff
@@ -0,0 +1,53 @@
+/*
+1. have lower and upper bound
+2. Guess the Number
+3. Ask higher or lower
+4. If correct: done, otherwise update bound, go back to step 1
+*/
+
+/*
+asyc wait is for input only
+if-else statements
+variable assignments
+if (guess <= 50) {
+  console.log("");
+} else {
+  console.log("denied");
+}
+use loops probably while loops to keep
+
+do I need variables for 100 guesses?
+I don't think so bc, I have asyc and while looping from 1-100
+I also have hugher and lower
```

Comment:
This is correct. Because you're using the while loop and updating the variables each time through you don't need to create new variables for each time.